firefox-ios/Client/Frontend/Browser/Tabs/Views/TabTrayViewController.swift

```diff
@@ -266,7 +266,9 @@ class TabTrayViewController: UIViewController,
         super.viewDidLoad()
         setupView()
         subscribeToRedux()
-        listenForThemeChange(view)
+        if !featureFlags.isFeatureEnabled(.feltPrivacySimplifiedUI, checking: .buildOnly) {
+            listenForThemeChange(view)
```

Comment:
the subviews were resetting with the current theme instead of following the tab tray panel. I kept the other tab tray panels as-is since I didn't see any issues

Reply:
I think there's one issue with this; if I leave the app to change the theme (in the iOS settings) and come back with the tab tray opened, we need the subviews to adapt to the new theme. Seems we need another way to have the private tab tray panel not to update its theme subviews.

[Video](https://github.com/user-attachments/assets/109f3c12-f7b0-4b8f-a56c-c74786b46a6a)

![Simulator Screenshot - iPhone 15 Pro - 2025-05-01 at 10 10 52](https://github.com/user-attachments/assets/baf855ee-d2dd-4c6f-8042-b6cd7346df34)